packages/core/helper-plugin/lib/src/content-manager/utils/formatContentTypeData.js

```diff
@@ -4,7 +4,7 @@
 import get from 'lodash/get';
 import { getType, getOtherInfos } from './getAttributeInfos';
 
-const formatComponentData = (data, ct, composSchema) => {
+const formatContentTypeData = (data, ct, composSchema) => {
```

Comment:
How do you feel about adding a new story in the storybook for this util? It could be great to document it WDYT?

Reply:
![Capture d’écran 2022-03-02 à 15 11 38](https://user-images.githubusercontent.com/11708220/156377905-c7c01a2a-c004-4745-8b96-11dc4ddd4310.png)

Yes :) How does it look like to you?